addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js

```diff
@@ -179,8 +179,13 @@ const PublicLivechatWindow = Widget.extend({
      * @private
      * @param {Object} messageData
      */
-    _postMessage(messageData) {
-        this.trigger_up('post_message_chat_window', { messageData });
+    async _postMessage(messageData) {
+        try {
+            await this.messaging.publicLivechatGlobal.livechatButtonView.sendMessage(messageData);
+        } catch (reason) {
+            reason.event.preventDefault();
```

Comment:
`reason.event`❓️
`Error.prototype.event` doesn't seem to exist; where is this property supposed to come from?

Reply:
No idea, good be worth just removing this LOC